integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -923,7 +923,7 @@ void testK8SEventsStartStopWatchingNSWithDedicated() {
     checkEvent(opNamespace, opNamespace, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
 
     logger.info("verify NamespaceWatchingStopped event is logged in {0}", domainNamespace4);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace4, null, "Normal", timestamp, false);
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace4, domainUid, "Normal", timestamp, false);
```

Comment:
domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID